### PR TITLE
doc update: karma.conf.js files array should reference coffee files

### PR DESCRIPTION
**Pull request by kylezeeuwen**
I got caught by this and have seen several others as well so thought it worth a note in the README.md.

The "dumb rookie" line is optional.

### CODE REVIEW

**Comment by kylezeeuwen** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ module.exports = function(config) {
       transformPath: function(path) {
         return path.replace(/\.coffee$/, '.js');
       }
+    },
+
+    // make sure to include the .coffee files not the compiled .js files 
+    // (they do not actually exist on the file system !)
+    // (you dumb rookie)
```
Comment (Dignifiedquire) [inline README.md]: Please remove this line, we don't want people reading this feeling insulted.

**Comment by Dignifiedquire** [comment]
Thanks, could you change the commit message so that it follows our [conventions](http://karma-runner.github.io/0.12/dev/git-commit-msg.html) please?

**Comment by kylezeeuwen** [comment]
I've updated the readme update and the commit message. If you would like me to add a (scope) to the commit message please advise what an appropriate scope would be.

**Comment by Dignifiedquire** [comment]
Thanks that's fine

**Pull request merged**